### PR TITLE
STY: conf.py updates

### PR DESCRIPTION
**Pull request by jklenzing**
# Description

Addresses #774 

- Improves maintainability of conf.py
- Updates style in conf.py to be flake8 compliant, add checks
- fixed a bug in the links

## Type of change

- Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

Inspection via read the docs

**Test Configuration**:
* Operating system: RTD
* Version number: ???
*  🐠 😼 🍣 

# Checklist:

- [x] Make sure you are merging into the ``develop`` (not ``main``) branch
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] Add a note to ``CHANGELOG.md``, summarizing the changes

If this is a release PR, replace the first item of the above checklist with the release
checklist on the wiki: https://github.com/pysat/pysat/wiki/Checklist-for-Release

### CODE REVIEW

**Comment by jklenzing** [comment]
Docs are online at: https://pysat.readthedocs.io/en/sty-conf-py/index.html

**Comment by jklenzing** [comment]
@aburrell, I've updated the style here basing it off of the release candidate at pysatSpaceWeather, as well as some things in the other packages (copyright, etc).

File: docs/conf.py [hunk 1]
```diff
@@ -255,32 +139,32 @@
...
 
 # -- Options for manual page output ---------------------------------------
 
 # One entry per manual page. List of tuples
 # (source start file, name, description, authors, manual section).
 man_pages = [
-    (master_doc, 'pysat', u'pysat Documentation',
+    (master_doc, project, title,
      [author], 1)
 ]
```
Comment (aburrell) [inline docs/conf.py:164]: This shouldn't be split over so many lines.

File: docs/conf.py [hunk 2]
```diff
@@ -289,23 +173,20 @@
 # (source start file, target name, title, author,
 #  dir menu entry, description, category)
 texinfo_documents = [
-  (master_doc, 'pysat', u'pysat Documentation',
-   author, 'pysat', 'One line description of project.',
-   'Miscellaneous'),
+    (master_doc, project, title, author, project, description,
+     'Space Physics'),
 ]
```
Comment (aburrell) [inline docs/conf.py:178]: Again, should take up less lines.

**Review by aburrell — APPROVED** [review]

**Pull request merged**